### PR TITLE
Change onboarding grammar and change username "anon" to "User"

### PR DESCRIPTION
**Pull request by pixl-8**
This will make it easier for users to understand the OS.

### CODE REVIEW

File: apps/onboarding/content3.markup [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
     <Container padding="insets(8, 0)" layout="vflow(2)">
         <Label text="Do you like skiftOS and want it to become even better?" anchor="center"/>
-        <Label text="Then there is four way to contribute to the project:" anchor="center"/>
-        <Label text="contributing code, reporting issues, donations and sponsorship" anchor="center"/>
+        <Label text="Then there are many ways to contribute to the project:" anchor="center"/>
+        <Label text="contributing code, reporting issues, donations and sponsorship." anchor="center"/>
```
Comment (notYuriy) [inline apps/onboarding/content3.markup:7]: `reporting issues, donations, and sponsorship.`
Add oxford comma here.

File: apps/onboarding/content1.markup [hunk 2]
```diff
@@ -3,8 +3,8 @@
 
     <Container padding="insets(8, 0)" layout="vflow(2)">
         <Label text="Click on the top left of the screen to bring up the application menu." anchor="center"/>
-        <Label text="There is also a terminal with familiar unix utilities" anchor="center"/>
-        <Label text="like ls, yes, grep, cd, which, ..." anchor="center"/>
+        <Label text="Checkout the terminal with familiar unix utilities" anchor="center"/>
```
Comment (notYuriy) [inline apps/onboarding/content1.markup:6]: `Checkout` should be written as `Check out`. I would suggest using `Try out` instead.

**Review by notYuriy — CHANGES_REQUESTED** [review]

File: sysroot/Configs/environment.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
     "POSIX": {
         "HOME": "/User",
-        "LOGNAME": "anon",
+        "LOGNAME": "User",
```
Comment (notYuriy) [inline sysroot/Configs/environment.json:4]: User name starting from capital case may be not very terminal friendly.

File: apps/onboarding/content2.markup [hunk 4]
```diff
@@ -2,8 +2,8 @@
     <Label text="Join The Community." anchor="center" />
 
     <Container padding="insets(8, 0)" layout="vflow(2)">
-        <Label text="The community is full of great peoples, join us on:" anchor="center"/>
-        <Label text="Github, Discord, and/or Twitter #skiftOS." anchor="center"/>
+        <Label text="The community is full of great people doing amazing things, join us now:" anchor="center"/>
+        <Label text="Github, Discord, and Twitter at #skiftOS." anchor="center"/>
```
Comment (notYuriy) [inline apps/onboarding/content2.markup:6]: Why `at` is used instead of `on`?

**Review by pixl-8 — COMMENTED** [review]
Sorry, I was in a rush when I checked this.

**Pull request closed without merging**